test/test_console.py

```diff
@@ -21,53 +18,39 @@ def check_write(value, expected, stream_encoding, preferred_encoding):
             locale.getpreferredencoding = lambda: preferred_encoding
 
             # Check writing to io.StringIO
-            if six.PY3:
-                stream = io.StringIO()
-                _write_with_fallback(value, stream)
```

Comment:
When i clean out PY2 lines this part breaks with 

```
test/test_console.py:27: Failed
============================================ short test summary info ============================================
FAILED test/test_console.py::test_write_with_fallback - Failed: DID NOT RAISE <class 'TypeError'>
```

Reply:
This is strange. Can you undo the specific changes that are causing the failure for now? Or if it's easier, just move the whole file with your changes to a new PR, and remove it from here. This way we can merge and forget what's working, and we can focus better on what's not.

Reply:
ok ,let me do this